services/promecieus/09-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: promecieus-robot-admin
+  namespace: promecieus
+subjects:
+  - kind: ServiceAccount
+    name: promecieus-robot
+    namespace: promecieus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
```

Comment:
I think that its better to create a `role` in your namespace with the required permissions and bind that instead of making your sa an `admin`.

Reply:
Good idea, created a dedicated role